### PR TITLE
feat: add new endpoint delete-flags

### PR DESCRIPTION
**Pull request by ms-fcabrera**
Jira ticket: [DAT-1240](https://makingsense.atlassian.net/browse/DAT-1240)

New endpoint to delete the application usage values

url: 
/{email}

Evidence:
GET
image
DELETE
image
GET
image

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -82,6 +83,12 @@ functions:
       - httpApi:
           path: /{email}/close-first-steps
           method: post
+  deleteFlags:
+    handler: src/deleteFlags.deleteFlags
+    events:
+      - httpApi:
+          path: /{email}/delete-flags
```
Comment (andresmoschini) [inline serverless.yml:90]: In my opinion, this is the right path:

```suggestion
          path: /{email}
```

What do you think?

Comment (ms-fcabrera) [reply serverless.yml:90]: Makes sense

**Comment by andresmoschini** [comment]
:shipit:

**Comment by jhoffman-ms** [comment]
👍

**Pull request merged**